src/containers/peacemaker/PeacemakerInformationForm.js

```diff
@@ -140,6 +147,20 @@ const PeacemakerInformationForm = ({ personId } :Props) => {
           onSubmit={onSubmit}
           schema={schema}
           uiSchema={uiSchema} />
+      {isSuccess(submitRequestState) && (
+        <CardSegment>
+          <SubmissionFieldsGrid>
+            <Typography gutterBottom>Submitted!</Typography>
+            <Button
+                aria-label="Success Button"
```

Comment:
According to MDN
> The `aria-label` attribute is used to define a string that labels the current element. Use it in cases where a text label is not visible on the screen. If there is visible text labeling the element, use `aria-labelledby` instead.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute